src/main/java/io/supertokens/webserver/api/session/JWTDataAPI.java

```diff
@@ -42,7 +43,7 @@ public String getPath() {
         return "/recipe/jwt/data";
     }
 
-    @Override
+    @Override @Deprecated
```

Comment:
The `put` API isn't deprecated right? Just the `GET` API... so you would also have to remove the `@Deprecated` from this class.

Reply:
Yeah youre right, shouldve noticed this. Done